models/DataObject/ClassDefinition/Data/Multiselect.php

```diff
@@ -359,6 +359,19 @@ public function getDataFromEditmode($data, $object = null, $params = [])
         return $data;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getDiffDataFromEditmode($data, $object = null, $params = []): ?array
+    {
+        $data = $data[0]['data'];
```

Comment:
I tried to merge events like described in the issue and get the follow exception
![image](https://user-images.githubusercontent.com/38959016/210531970-e875689d-8d4a-413f-bfd4-17030d002d35.png)
Seems like the input param $data is already a string

Reply:
Yeah my bad sorry :) Somehow it was not applied correctly.